sdks/java/core/src/main/java/org/apache/beam/sdk/util/HistogramData.java

```diff
@@ -573,6 +613,40 @@ public double getRangeTo() {
     // Note: equals() and hashCode() are implemented by the AutoValue.
   }
 
+  // Used for testing unsupported Bucket formats
+  @AutoValue
+  public abstract static class UnsupportedBuckets implements BucketType {
```

Comment:
nit: format comment like javadoc

`/** Used for testing unsupported Bucket formats. */`